disassemblers/ofrak_ghidra/ofrak_ghidra/components/ghidra_analyzer.py

```diff
@@ -220,6 +225,51 @@ def _build_ghidra_server_args(self) -> List[str]:
 
         for ghidra_script in self._scripts:
             args.append(ghidra_script)
-            args.append(ghidra_script.split(".")[0].lower())
 
         return args
+
+
+@dataclass
+class GhidraCodeRegionConfig(ComponentConfig):
+    pass
+
+
+class GhidraCodeRegionModifier(Modifier[Optional[GhidraCodeRegionConfig]], OfrakGhidraMixin):
+    id = b"GhidraCodeRegionModifier"
+    targets = (CodeRegion,)
+
+    get_code_regions_script = OfrakGhidraScript(
+        os.path.join(CORE_OFRAK_GHIDRA_SCRIPTS, "GetCodeRegions.java"),
+    )
+
+    async def modify(self, resource: Resource, config: Optional[GhidraCodeRegionConfig] = None):
```

Comment:
Components don't need to define a config if they don't take any "parameter"

```suggestion

class GhidraCodeRegionModifier(Modifier[None], OfrakGhidraMixin):
    id = b"GhidraCodeRegionModifier"
    targets = (CodeRegion,)

    get_code_regions_script = OfrakGhidraScript(
        os.path.join(CORE_OFRAK_GHIDRA_SCRIPTS, "GetCodeRegions.java"),
    )

    async def modify(self, resource: Resource, config=None):
```